flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -137,11 +137,11 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
         'ui.load',
         bindToScope: scope.span == null,
         waitForChildren: true,
+        autoFinishAfter: Duration(seconds: 3),
```

Comment:
let's expose this configurable via SentryOptions?

Reply:
Added #667 for this.